Kbs.Wpf/Game/Read/Index/ReadIndexGameGameViewModel.cs

```diff
@@ -46,4 +45,10 @@ public string Course
         get => _course;
         set => SetField(ref _course, value);
     }
+
+    public string DateString
+    {
+        get => Date.ToDutchString();
+    }
```

Comment:
Moet geüpdatet worden in de Date setter. Kijk af van: `ReadDamageDetailsViewModel.Status` (roept `StatusFormatted` aan als veranderde property)